src/regtech_data_validator/validation_results.py

```diff
@@ -10,10 +10,17 @@ class ValidationPhase(StrEnum):
 
 
 @dataclass(frozen=True)
-class ValidationResults(object):
+class Count(object):
     single_field_count: int
     multi_field_count: int
     register_count: int
+    total_count: int
```

Comment:
maybe default these to 0, so the above code can just instantiate it with no args

Reply:
clarification, so counts can be instantiated as `error_counts = warning_counts = Count()`

Reply:
@lchen-2101 Is this still necessary since the `error_counts` and `warning_counts` will be set to `None` initially?

Reply:
uh... they aren't being set to `None`... `error_counts = warning_counts = Counts()`